### ISSUE TITLE
Gehstil wird durch schleichen zurückgesetzt

### ISSUE THREAD

**Issue by FunnyBuddys** [mexyn/statev_v2_issues#1689]
**Character Name**
Ranjid_Boniakale

**Auftrittszeitpunkt (Datum / Uhrzeit)**
12.07.2021; 20:35

**Beobachtetes Verhalten**
Gehstil wurde nach dem Schleichen zurückgesetzt.

**Erwartetes Verhalten**
Gehstil bleibt auch nach dem Schleichen.

**Schritte um den Fehler nachvollziehen zu können**
Einen X-Beliebigen Gehstil auswählen (außer natürlich der Standart-Gehstil) dann über den Zirkumflex (^) ins schleichen übergehen und das schleichen dann wieder beenden. Anschließend ist der Gehstil wieder Standart obwohl man diesen nicht ausgewählt hat.

**Comment by DonVitoDiangelo** [comment]
Ja, du hast ja zwischendrin deinen Gehstil geändert, indem du geschlichen bist und gehst dann wieder auf normal.
Mal gucken ob wir da was machen. @its4l3x

**Comment by its4l3x** [comment]
Ja das ist so gewollt. Ist ein enhancement für die Zukunft.
